client/my-sites/site-settings/disconnect-site/disconnect-follow-up.jsx

```diff
@@ -0,0 +1,162 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import SelectDropdown from 'components/select-dropdown';
+import { isFreeJetpackPlan, isPersonal } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectFollowUp extends Component {
+	state = {
+		reasonSelected: 'Backups',
+	};
+
+	logReasonFollowUp = option => {
+		this.setState( {
+			reasonSelected: option.value,
+		} );
+	};
+
+	setOptionsPaidPlan = () => {
+		const { site } = this.props;
+
+		let options = [];
+		options = [
+			{ value: 'Backups', label: 'Backups' },
+			{ value: 'Antispam', label: 'Antispam' },
+			{ value: 'Stats', label: 'Stats' },
+			{ value: 'Publicize', label: 'Publicize' },
+			{ value: 'Subscriptions', label: 'Subscriptions' },
+			{ value: 'Other', label: 'Other' },
+		];
+		if ( ! isPersonal( site.plan ) ) {
+			options.push( { value: 'Security Scanning', label: 'Security Scanning' } );
+		}
+
+		return options;
+	};
+
+	getFollowUpQuestion = optionValue => {
+		const { site, translate } = this.props;
+
+		let textShareWhat = '';
+
+		switch ( optionValue ) {
+			case 'tooHard':
+				if ( isFreeJetpackPlan( site.plan ) ) {
+					textShareWhat = translate( 'Can we help ? ' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'How can we help?' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Questions/ notes :
-  should we remove `Skip Survey` in the last screen?There is a way out of the Flow already,
or keep it for consistency?

- `What features were you looking for?` is meant as an autocomplete search, so the component will resemble the Search action for `Themes`. Here the simple search is a placeholder.

Reply:
but that might be a better choice of a questions, actually :)